docs/components/query-frontend.md

```diff
@@ -36,7 +36,7 @@ Query Frontend supports a retry mechanism to retry query when HTTP requests are
 
 Query Frontend supports caching query results and reuses them on subsequent queries. If the cached results are incomplete, Query Frontend calculates the required subqueries and executes them in parallel on downstream queriers. Query Frontend can optionally align queries with their step parameter to improve the cacheability of the query results. Currently, in-memory cache (fifo cache) and memcached are supported.
```

Comment:
Might be not related to this pr. But QF supports redis cache now. It would be good if you can update this as well.